scripts/logic/test/env_test.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+export image_id=$1
+export image_full_tag=$2
+
+export image_full_name=${image_full_tag%%:*}
+export image_full_name=${image_full_name%%@sha256*} # Remove `@sha256` is needed
```

Comment:
Probably you meant
```suggestion
export image_full_name=${image_full_name%%@sha256*} # Remove `@sha256` if needed
```
?